src/implicit.cc

```diff
@@ -240,6 +228,8 @@ namespace hpp {
         parameterSize_ = computeParameterSize (comparison_);
       ar & BOOST_SERIALIZATION_NVP(function_);
       ar & BOOST_SERIALIZATION_NVP(rhsFunction_);
+      ar & BOOST_SERIALIZATION_NVP(activeRows_);
+      ar & BOOST_SERIALIZATION_NVP(inactiveRows_);
```

Comment:
For compactness of the archive,
```suggestion
      if (!Archive::is_saving::value)
        computeActiveRows();
```